objects/client.c

```diff
@@ -577,7 +577,7 @@
  */
 
 /**
- * The client the window is transient for.
+ * The client is transient for.
```

Comment:
The old definition was terrible, but this one isn't very useful/better. `transient for` is another `client` object.

For example, a file open dialog can be transient for its parent client